cloudwatchlogs/cloudwatchlogs_lambda.js

```diff
@@ -8,7 +8,7 @@ exports.handler = function(event, context) {
 		///////////////////////////////////////////////////////////////////////////////////////////////////////////
 		var options = {
 			'hostname': 'collectors.sumologic.com',
-			'path': 'https://collectors.sumologic.com/receiver/v1/http/<XXX>',
+			'path': 'receiver/v1/http/<XXX>',
```

Comment:
Thanks. The path should actually start with "/". Also since we are using https, probably better to include 'procotol' property as well.